src/util/analytics-client.ts

```diff
@@ -0,0 +1,92 @@
+import {getLogger} from "../config/logger";
+import { omit } from "lodash";
+import { optionalRequire } from "optional-require";
+import {isNodeProd} from "utils/is-node-env";
+
+const { analyticsClient } = optionalRequire("@atlassiansox/analytics-node-client") || {};
+const logger = getLogger("analytics")
+const instance = process.env.INSTANCE_NAME;
+const appKey = `com.github.integration${instance ? `.${instance}` : ""}`;
+
+let analyticsNodeClient;
+
+function sendAnalytics(eventType: "trait")
+function sendAnalytics(eventType: "screen", attributes:{name:string} & Record<string,unknown>)
+function sendAnalytics(eventType: "ui" | "track" | "operational", attributes: Record<string,unknown>)
+function sendAnalytics(eventType: string, attributes?: Record<string, unknown>): void {
+	if(!analyticsClient && !isNodeProd()){
+		return;
+	}
+
+	if(!analyticsNodeClient){
```

Comment:
Looks like analyticsNodeClient is always undefined at this point?
Could remove the condition if thats the case?